modules/library/render/src/test/java/org/geotools/renderer/lite/AbstractLabelLineTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2016, Open Source Geospatial Foundation (OSGeo)
```

Comment:
The file is actually a copy and refactor of a tests @nmco made in 2016, so that's the right date.